frontend/src/utils/CreateEmailLink.ts

```diff
@@ -0,0 +1,48 @@
+import parse from './TemplateParser'
+export {}
+
+function assertEqual(expect: any, expression: any) {
+  if (expect !== expression) {
+    throw new Error(`AssertionFailure: expected ${expect}, got ${expression}`)
+  }
+}
+
+function constructMailtoUrl(
+  receivers: string[],
+  subject: string,
+  body: string
+): string {
+  let url = 'mailto:'
+  url += receivers.join(',')
+  url += '?subject=' + subject
+  url += '&body=' + encodeURI(body)
```

Comment:
I liked how you split the concatenations into several lines for readability. Maybe you don't need to do this 3 times though—you could combine it in one multi-line statement?